tests/unit_test.sh

```diff
@@ -1,8 +1,9 @@
 #!/bin/bash
 # This is for use as a Github CI Unit Test.
-# Version 1.3
+# Version 1.4
 set -e
 cd /usr/local/lib/crew/packages/
+CREW_BRANCH="${CREE_BRANCH}" crew update && yes | cree upgrade
```

Comment:
`crew upgrade`?

Reply:
`CREW_BRANCH="${CREE_BRANCH}"` shouldn't be necessary. `CREE_BRANCH`?

Reply:
Sigh. Yes. I should increase the font size on my phone ...